app/i18n/en/sub.php

```diff
@@ -121,6 +121,45 @@
 				'relative' => 'XPath (relative to item) for:',
 				'xpath' => 'XPath for:',
 			),
+			'json_dotpath' => array(
+				'_' => 'JSON (Dotted paths)',
```

Comment:
`Dotted paths` => would be `Dot-Notation` the correct term here?

(ping @Alkarex @Frenzie )

Reply:
Dot notation is the term I'm familiar with, yes.

Reply:
Back when we had the PR, I tried to search for standard / usual names. I could not find much, so references welcome

Reply:
something like https://docs.oracle.com/en/database/oracle/oracle-database/19/adjsn/simple-dot-notation-access-to-json-data.html ?

Reply:
Dot notation is what it's called in JS and Java (and probably other languages as well), as distinguished from this['is']['equivalent']['bracket']['notation'].

It's not related to JSON per se of course, just to how you access object values.

Reply:
i18n Strings improved: #6317